__tests__/app.test.js

```diff
@@ -26,4 +29,36 @@ describe('/api/topics', () => {
             })
         })
     })
+})
+
+describe('/api/articles/:id', () => {
+    describe('GET', () => {
+        test('status: 200 - responds with an article object depending on ID of article.', () => {
+            return request(app).get('/api/articles/1').expect(200).then((response) => {
+                    expect(response.body.article).toMatchObject({
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number)
+                    })
+                    expect(response.body.article).toEqual({
+                        article_id: 1,
+                        title: 'Living in the shadow of a great man',
+                        topic: 'mitch',
+                        author: 'butter_bridge',
+                        body: 'I find this existence challenging',
+                        created_at: '2020-07-09T20:11:00.000Z',
+                        votes: 100
+                    })
```

Comment:
brittle testing, you may be adding an extra key at some point in which this test will fail 👀